integration-cli/docker_cli_daemon_test.go

```diff
@@ -884,8 +884,10 @@ func TestDaemonwithwrongkey(t *testing.T) {
 
 	if err := d1.Start(); err == nil {
 		d1.Stop()
+		os.Remove("/etc/docker/key.json")
```

Comment:
Instead of having two `os.Remove` calls, could we maybe have a single `defer os.Remove("/etc/docker/key.json")` on line 882 (right after the file gets written)? Thanks!